### PR TITLE
Implemented DelegatingOutputFormat for schemaless GCS multisink

### PR DESCRIPTION
**Pull request by fernst**

### CODE REVIEW

File: src/main/java/io/cdap/plugin/gcp/gcs/sink/DelegatingGCSOutputCommitter.java [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+  public void commitJob(JobContext jobContext) throws IOException {
+    for (OutputCommitter committer : committerMap.values()) {
+      committer.commitJob(jobContext);
+    }
+  }
+
+  @Override
+  public void abortTask(TaskAttemptContext taskAttemptContext) throws IOException {
+    for (OutputCommitter committer : committerMap.values()) {
+      committer.abortTask(taskAttemptContext);
```
Comment (albertshau) [inline src/main/java/io/cdap/plugin/gcp/gcs/sink/DelegatingGCSOutputCommitter.java]: if one abort throws an exception, we still want to try to abort the others.

File: src/main/java/io/cdap/plugin/gcp/gcs/sink/DelegatingGCSOutputCommitter.java [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+ */
+public class DelegatingGCSOutputCommitter extends OutputCommitter {
+  private final Map<String, OutputCommitter> committerMap;
+
+  public DelegatingGCSOutputCommitter() {
+    committerMap = new HashMap<>();
+  }
+
+  /**
+   * Add a new GCSOutputCommitter based on a supplied Input Format and Table Name.
```
Comment (albertshau) [inline src/main/java/io/cdap/plugin/gcp/gcs/sink/DelegatingGCSOutputCommitter.java]: Input -> Output

Comment (fernst) [reply src/main/java/io/cdap/plugin/gcp/gcs/sink/DelegatingGCSOutputCommitter.java]: Fixed.

File: src/main/java/io/cdap/plugin/gcp/gcs/sink/DelegatingGCSOutputCommitter.java (hunk 1 above)
Comment (fernst) [reply src/main/java/io/cdap/plugin/gcp/gcs/sink/DelegatingGCSOutputCommitter.java]: Fixed.

**Review by albertshau — APPROVED** [review]

**Pull request merged**